runtime/makelib/targets.mk.osx.inc.ftl

```diff
@@ -47,7 +47,7 @@ $(UMA_EXETARGET): $(UMA_OBJECTS) $(UMA_TARGET_LIBRARIES)
 UMA_BEGIN_DASH_L =
 UMA_END_DASH_L =
 
-UMA_EXE_POSTFIX_FLAGS += -lm -liconv -lc -ldl -lutil -Wl,-rpath,\$$ORIGIN
+UMA_EXE_POSTFIX_FLAGS += -lm -liconv -lc -ldl -lutil -Wl,-rpath,@loader_path/.
```

Comment:
Why `/.` on the end of the path?

Reply:
i think it should work without `/.`. I will verify and remove it.

Reply:
verified; works without `/.`; removed it.